build.gradle

```diff
@@ -1,4 +1,10 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
+buildscript {
+    dependencies {
+        classpath 'io.github.gradle-nexus:publish-plugin:1.1.0'
+    }
+}
+
```

Comment:
nit: this is another way to add plugins to the project, since we have some using the plugins block, we can move it:

```
plugins {
    id 'com.android.application' version '7.3.1' apply false
    id 'com.android.library' version '7.3.1' apply false
    id 'org.jetbrains.kotlin.android' version '1.7.10' apply false
    id 'org.jetbrains.kotlin.plugin.serialization' version '1.7.10'
    id "com.diffplug.spotless" version "6.11.0"
    id 'androidx.navigation.safeargs' version '2.5.3' apply false
    id 'com.google.dagger.hilt.android' version '2.44' apply false
    id 'io.github.gradle-nexus.publish-plugin' version '1.1.0'
}
```

and remove `apply plugin: 'io.github.gradle-nexus.publish-plugin'`